server/api/index.js

```diff
@@ -114,4 +114,19 @@ router.delete("/organisations/:id", async (req, res) => {
   }
 });
 
+router.put("/organisations/:organisation_id", async (req, res) => {
+  const organisation_id = `${req.params.organisation_id}`;
```

Comment:
You can directly use req.params.organisation_id here, no need to have quotes and string interpolation.